src/persistence/recordings.ts

```diff
@@ -1,36 +1,11 @@
 import fs from "fs";
 import path from "path";
-import { config } from "../config/config";
 import { MoujiRecording } from "../types/MoujiRecording";
 import { info, verbose } from "../utils/logger";
 import { deserializeRecording, serializeRecording } from "../utils/recording";
 
-function ensureMoujiTempExists(): void {
-  if (!fs.existsSync(config.tmpPath)) {
-    verbose(`Creating ${config.tmpPath}`);
-    fs.mkdirSync(config.tmpPath);
-  }
-}
-
-function ensureMoujiRecordingExists(): void {
-  ensureMoujiTempExists();
-  if (!fs.existsSync(config.recordPath)) {
-    verbose(`Creating ${config.recordPath}`);
-    fs.mkdirSync(config.recordPath);
-  }
-}
-
-export function list(): Array<string> {
-  ensureMoujiRecordingExists();
-
-  return fs
-    .readdirSync(config.recordPath)
-    .filter((file) => file.endsWith(".moujirec"))
-    .map((file) => file.replace(".moujirec", ""));
-}
-
 export function load(recording: string): MoujiRecording {
-  const file = path.join(config.recordPath, `${recording}.moujirec`);
+  const file = path.join(process.cwd(), `${recording}.moujirec`);
```

Comment:
I think you may want to prefer `path.resolve` here so that we don't impose an artificial limitation on the path option we're allowed to supply

I'd also do this up a layer and not strictly apply the extension like this, bc I'm probably going to include the extension anyway when I specify the arg myself

Reply:
Ah yeah you can tell i didn't test this at all